pandas/core/format.py

```diff
@@ -1795,18 +1791,14 @@ def _format_regular_rows(self):
             else:
                 index_label = self.df.index.names[0]
 
+            if isinstance(self.columns, MultiIndex):
+                self.rowcounter += 1
+
             if index_label and self.header is not False:
-                if self.merge_cells:
```

Comment:
It is, there is still the non-default option to write the MI as non merged cells, it just no longer effects this particular offset.